tests/utils/verification/volumeproperties.go

```diff
@@ -161,3 +164,25 @@ func GetAssociatedPolicyName(hostname string, volName string) (string, error) {
 	}
 	return op, err
 }
+
+// GetVMGroupForVolume returns vmgroup field of volume using admin cli
+// If the volume does not exist, err will be filled with "exit status 1"
+func GetVMGroupForVolume(hostName string, volName string) (string, error) {
+	cmd := admincli.ListVolumes + "-c volume,vmgroup 2>/dev/null | grep " + volName
+	op, err := ssh.InvokeCommand(hostName, cmd)
+	if err != nil {
+		log.Printf("GetVMGroupForVolume return with err: %s", err.Error())
+		return "", err
+	}
+	volProps := strings.Fields(op)
+	if op == "" {
+		msg := fmt.Sprintf("Null value is returned by admin cli when looking for vmgroup field for volume %s", volName)
+		log.Printf(msg)
+		return op, errors.New(msg)
+	}
+	if len(volProps) != 2 {
+		log.Printf("Admin cli output is expected to consist of two elements only - "+
+			"volume name and vmgrop. Actual output %s ", op)
```

Comment:
The volume name used in test is unique since it use random generator. So  it is not very possible that we hit the scenario that we have same volume name with two different datastores.

For a orphan volume, ``` docker volume inspect ``` may not able to get the vmgroup associated with the volume.